functions/Add-DbaComputerCertificate.ps1

```diff
@@ -78,36 +75,39 @@ Adds the local C:\temp\cer.cer to the local computer's LocalMachine\My (Personal
 			return
 		}
 		
-		if (!(Test-Path -Path $path)) {
-			Write-Message -Level Warning -Message "$Path does not exist"
-			return
+		if ($Path) {
+			if (!(Test-Path -Path $Path)) {
+				Write-Message -Level Warning -Message "$Path does not exist"
+				return
+			}
 		}
 		
-		foreach ($computer in $computername) {
-			if (![dbavalidate]::IsLocalhost($computer) -and !$Password) {
-				Write-Message -Level Output -Message "You have specified a remote computer. A password is required for private key encryption/decryption for import."
-				$Password = Read-Host -AsSecureString -Prompt "Password"
+		if (![dbavalidate]::IsLocalhost($computer) -and !$Password) {
```

Comment:
Ouu, i like $computer.IsLocalhost - ohh, i need to check that. I moved it around.